src/org/thoughtcrime/securesms/ProfileSettingsAdapter.java

```diff
@@ -261,10 +266,13 @@ public void changeData(@Nullable int[] memberList, @Nullable DcContact dcContact
     itemDataMemberCount = 0;
     itemDataSharedChats = null;
     itemDataContact = null;
+    itemDataChat    = null;
 
     if (memberList!=null) {
       itemDataMemberCount = memberList.length;
-      if (!dcChat.isMailingList()) {
+      if (dcChat.isMailingList()) {
+        itemDataChat = dcChat;
```

Comment:
What do we need this variable for?

Reply:
it is used at https://github.com/deltachat/deltachat-android/pull/1772/files#diff-d483d329b19c7c242afce2368bf6fcf6acbe10a21c80996b74831eaf99cba30bR219 to set a different subtitle - "X members" is wrong for mailing lists, in fact, we do not know the number of members.

`itemDataChat` is a bit weird, however, i am just following the style already used in the file :)

Reply:
It somehow confuses me that `isMailingList()` is checked twice (and generally, that we only have an `itemDataChat` if we have a mailing list) :) Maybe just unconditionally set it here - `isMailingList()` is [checked above](https://github.com/deltachat/deltachat-android/pull/1772/files#diff-d483d329b19c7c242afce2368bf6fcf6acbe10a21c80996b74831eaf99cba30bR219), anyway - or make a `bool` `isMailingList`?

Reply:
yip, we can do a `isMailingList` list, that is probably more straight-forward. i'll add a commit.